### PR TITLE
add laravel queue redis module

### PR DESCRIPTION
**Pull request by halaei**
https://github.com/halaei/lqrm
related issues:
https://github.com/laravel/framework/issues/22939
https://github.com/antirez/redis/issues/2801

### CODE REVIEW

**Comment by tomekit** [comment]
Hi @halaei 
Did you find any alternative reliable solution to queueing?
I've got this so far: https://gist.github.com/tomekit/676e93e67c0fe45d2d7f576bc0672542 but it doesn't support long-polling unfortunately.

**Comment by halaei** [comment]
@tomekit As of Laravel 5.8 blocking pop is reliable (see the related pull request https://github.com/laravel/framework/pull/27336 and ). So the main advantage of my lqrm module now is blocking pop from delayed queues, which is still impossible in Larvel's redis driver. Please note I don't feel 100% confidence to advice using lqrm for production yet.

File: modules.json [hunk 1]
```diff
@@ -185,5 +185,15 @@
...
     ],
     "stars": 613
+  },
+  {
+    "name": "lqrm",
+    "license": "BSD",
+    "repository": "https://github.com/halaei/lqrm",
+    "description": "A Laravel compatible queue driver for Redis that supports reliable blocking pop from FIFO and scheduled queues.",
+    "authors": [
+      "Hamid Alaei Varnosfaderani"
```
Comment (itamarhaber) [inline modules.json]: This needs to be a valid Twitter (not GitHub) handle - it is ok to leave empty if you don't have one.

**Comment by itamarhaber** [comment]
Thanks - I took the liberty to fix the author field (feel free to open a new PR if you want your Twitter handle there) and star count. Merging.

**Comment by halaei** [comment]
@itamarhaber Thanks for accepting my PR :) I added my github username in other PR.

**Pull request merged**